File: src/main/java/com/hedvig/backoffice/services/account/dto/AccountDTO.java
package com.hedvig.backoffice.services.account.dto;

import lombok.Value;

import javax.money.MonetaryAmount;
import java.util.List;

@Value
public class AccountDTO {
  String memberId;
  MonetaryAmount currentBalance;
  MonetaryAmount totalBalance;
  List<AccountEntryDTO> entries;
}

File: src/main/java/com/hedvig/backoffice/services/account/dto/AccountDTO.kt
package com.hedvig.backoffice.services.account.dto

import javax.money.MonetaryAmount

data class AccountDTO(
    val memberId: String,
    val currentBalance: MonetaryAmount,
    val totalBalance: MonetaryAmount,
    val entries: List<AccountEntryDTO>,
    val chargeEstimation: AccountChargeEstimationDTO
)
